_pages/outreach.html

```diff
@@ -118,6 +118,17 @@ <h5>AUVSI House of Representatives Exhibit</h5>
         <img src="/assets/img/rep_3.JPG" alt="The team huddling around the robot.">
         <img src="/assets/img/rep_4.JPG" alt="A crowd including team members and others around a table.">
     </div>
+		<h5>Erik Boesen Testimony</h5>
+			<p>On the 29th January 2018, our own Erik Boesen--along with six members of FRC team 1086--spoke in favor of bill HB-1111 which would establish the STEAM (Science, Technology, Engineering, Arts, Mathemaitcs) Education Fund. As the name denotes, this fund would award grants to primary and secondary schools in the Commonwealth of Virginia.</p>
```

Comment:
— not --
Team not team
Bill not bill
HB1111 not HB-1111
Mathematics not Mathemaitcs

Try this:

On January 29th and February 7th, 2018, our programming captain Erik Boesen spoke in the Virginia General Assembly in support of HB1111, a bill we helped propose to provide funding for STEAM programs in underfunded Virginian schools through a public-private partnership.